graphql_compiler/tests/test_compiler.py

```diff
@@ -2145,6 +2145,96 @@ def test_contains_op_filter_with_optional_tag(self):
 
         check_test_data(self, test_data, expected_match, expected_gremlin, expected_sql)
 
+    def test_not_contains_op_filter_with_variable(self):
+        test_data = test_input_data.contains_op_filter_with_variable()
+
+        expected_match = '''
+            SELECT
+                Animal___1.name AS `animal_name`
+            FROM (
+                MATCH {{
+                    class: Animal,
+                    where: ((alias NOT CONTAINS {wanted})),
+                    as: Animal___1
+                }}
+                RETURN $matches
+            )
+        '''
+        # "NOT" is not supported? https://orientdb.com/docs/2.0/orientdb.wiki/SQL-Where.html
```

Comment:
I don't think it's supported like this -- this would make `NOT CONTAINS` an operator. `NOT` is supported as a general unary operator, so in this case you'd use it as `NOT (alias CONTAINS {wanted})`.

Reply:
aha, I found it!